neurodamus/cell_distributor.py

```diff
@@ -502,7 +506,7 @@ def getSpGid(self, gid):
         """
         manager = self._find_manager(gid)
         if manager._binfo:
-            return manager._binfo.thishost_gid(gid)
+            return manager._binfo.thishost_gid(gid - manager.local_nodes.offset)
```

Comment:
I wonder why we don't do `+ gid_offset` like 10 lines above. I don't racall if this should do the same, just noticing it seems inconsistent and if you know the reason for the difference maybe let's write a comment

Reply:
You are right, I probably forgot because I think this function is not used...